Blaze/ByteString/Builder/Internal/Write.hs

```diff
@@ -126,9 +126,6 @@ getBound' msg write =
 instance Semigroup Poke where
```

Comment:
The conditional API via `#if MIN_VERSION_base(4,9,0)` can be avoided since you added a dependency on `semigroup`

Reply:
You're right. I have removed conditionals here.